db/migrate/20220426233640_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] [REQUIRED] Kindly note that the `User` table according to the ERD diagram has the `posts_counter` . Kindly implement this.